dags/openshift_nightlies/scripts/install/hypershift.sh

```diff
@@ -102,9 +124,8 @@ create_empty_cluster(){
     echo "Create None type Hosted cluster.."    
     export NETWORK_TYPE=$(cat ${json_file} | jq -r .hosted_cluster_network_type)
     export REPLICA_TYPE=$(cat ${json_file} | jq -r .hosted_control_plane_availability)   
-    BASEDOMAIN=$(_get_base_domain $(_get_cluster_id ${MGMT_CLUSTER_NAME}))
     echo $PULL_SECRET > pull-secret
-    hypershift create cluster none --name $HOSTED_CLUSTER_NAME --node-pool-replicas=0 --base-domain $BASEDOMAIN --pull-secret pull-secret --control-plane-availability-policy $REPLICA_TYPE --network-type $NETWORK_TYPE
+    hypershift create cluster none --name $HOSTED_CLUSTER_NAME --node-pool-replicas=0 --base-domain $BASEDOMAIN --pull-secret pull-secret --control-plane-availability-policy $REPLICA_TYPE --network-type $NETWORK_TYPE --control-plane-operator-image=quay.io/hypershift/hypershift:latest
```

Comment:
We should probably add an option to point to a control-plane-operator-image based on a variable as well.